test/parallel/test-stream-readable-async-iterators.js

```diff
@@ -396,6 +396,30 @@ async function tests() {
     }
   }
 
+  {
+    console.log('readable side of a transform stream pushes null');
```

Comment:
would prefer to omit `console.log()` statements that aren't essential to the test. Perhaps make it a comment or change to use `debug()` statements using `util.debuglog()`

```suggestion
    // Readable side of a transform stream pushes null
```

Reply:
this follows the pattern of the rest of the test file.. should we turn them all into comments?

Reply:
There are differing opinions on this. I don't like `console.log()` statements in tests that aren't part of the actual test. If not comments, then maybe add a `const { debuglog } = require('util'); const debug = debuglog('test');` at the top and turn these into `debug('...')` statements. Our CI environment does not understand that but if someone wants to run the tests locally with `NODE_DEBUG=test` to optionally see the output then they can.

Reply:
These are my fault. I'd recommend to:

1. keep the current style
2. change the style in a follow up PR.